src/include/hook.h

```diff
@@ -101,8 +101,9 @@ typedef enum hook_user hook_user;
 #define HOOK_EVENT_EXECHOST_STARTUP	0x2000
 #define HOOK_EVENT_EXECJOB_ATTACH	0x4000
 #define HOOK_EVENT_EXECJOB_RESIZE	0x20000
+#define HOOK_EVENT_EXECJOB_ABORT	0x40000
```

Comment:
Was wondering why there were gaps in the numbers until I looked at the section above that defines the server hooks. Just a suggestion... it might be better to combine these two sections and add a comment to the end of each line indicating whether it is a server or mom hook. That way they will all be in order.

Reply:
@mike0042: Thanks for the suggestion, but I'll leave it as is.